File: java/src/poop_project/Selection.java
package poop_project;

public class Selection extends Rectangle{

	boolean active = true;
	int X;
	int Y;
	int H;
	int W;

	public Selection(int w,int h,int xx, int yy, int a, int b, int c, int d) {
		super(w, h, xx, yy);
		X = a;
		Y = b;
		W = c;
		H = d;
	}

	

}

File: cpp/Selection.cpp
#include"Selection.h"

Selection::Selection() {}
Selection::Selection(const std::string ime,std::vector<Rectangle> vec):name(ime),selected(vec) {}

const std::string& Selection::getName()const { return name; }
std::vector<Rectangle> Selection::getSelected()const { return selected; }

void Selection::save(const std::string& file) {

}